samples/system-test/inspect.test.js

```diff
@@ -53,7 +53,11 @@ it('should inspect a string', async () => {
     `${cmd} string "I'm Gary and my email is gary@example.com"`,
     cwd
   );
-  assert.strictEqual(new RegExp(/Info type: EMAIL_ADDRESS/).test(output), true);
+  assert.strictEqual(
```

Comment:
For cases like this, it's a lot nicer if we can `npm install --save-dev chai`, and use `const {assert} = require('chai')`.  That way, you can use `assert.matches()`.  The benefit is that when the test fails, you get the printout of the regex and the value instead of just `false !== true`.

Reply:
Seems easy enough. I'm not familiar with Node or with how this repo works. Do I need to change any config files to indicate that chai should be installed?